src/docs/reference/modules/Security/README.md

```diff
@@ -13,6 +13,9 @@ Enabling the `OrchardCore.Security` module will allow the user to set the follow
 | `PermissionsPolicy` | Gets or sets the `Permissions-Policy` HTTP header. |
 | `ReferrerPolicy` | Gets or sets the `Referrer-Policy` HTTP header. |
 
+!!! note
+    The `Content-Security-Policy` HTTP header contains `frame-ancestors` directive which obsoleted `X-Frame-Options` HTTP header.
```

Comment:
I would suggest adding an example like setting `Frame Ancestors` option to `self` or something like that.

Reply:
For more details, I suggest it should redirect to the HTTP headers spec